src/app/components/builder/templates/one-page-standard/logic/headers/page.header.logic.ts

```diff
@@ -27,7 +27,13 @@ export function constructHeader(
   /** Construct Additional Details */
   cursor.setSize(standard.TEXT_FONT_SIZE);
   updateFontAndSize(jsPDFInstance, standard.FONT_NAME, FontStyle.REGULAR, cursor.getSize());
-  const contactInformation = `${header.leftDetail} | ${header.email} | ${header.phone} | ${header.rightDetail}`;
+  let contactInformation = `${header.email} | ${header.phone}`;
+  if (header.leftDetail) {
+    contactInformation = `${header.leftDetail} | ${contactInformation}`;
+  }
+  if (header.rightDetail) {
+    contactInformation = `${contactInformation} | ${header.rightDetail}`;
+  }
```

Comment:
eventually rightDetail will overwrite left details, is this the expect behaviour?

Reply:
It will not override, just append to the right of the contactInformation variable :)